disko

```diff
@@ -133,7 +133,7 @@ if [[ -n "${flake+x}" ]]; then
   nix_args+=("--argstr" "flakeAttr" "$flakeAttr")
   nix_args+=(--extra-experimental-features flakes)
 elif [[ -n "${disko_config+x}" ]] && [[ -e "$disko_config" ]]; then
-  nix_args+=("--arg" "diskoFile" "$disko_config")
+  nix_args+=("--arg" "diskoFile" $(realpath "$disko_config"))
```

Comment:
```suggestion
  nix_args+=("--arg" "diskoFile" "$(realpath "$disko_config")")
```

Reply:
Ah, good point, paths can contain spaces 😅 I also fixed this for `$flake`, it wasn't quoted either.